client/components/ProjectHeading.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components/native';
+import { AntDesign } from '@expo/vector-icons';
+
+
+const ProjectHeading = () => {
+
+    const ProjectHeadingView = styled.View`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    `
+
+    const ProjectTitle = styled.Text`
+    font-weight: 700;
+    font-size: 18px;
+    `
```

Comment:
these styled components should be outside of the component definition as they need to be static